doc/operations/ledger_snapshot.rst

```diff
@@ -86,3 +86,18 @@ A low value for ``ledger.chunk_size`` means that smaller ledger files are genera
 Similarly, a low value for ``snapshots.tx_count`` means that snapshots are generated often and that join/recovery time will be short, at the cost of additional workload on the primary node for snapshot generation.
 
 .. tip:: Uncommitted ledger files (which are likely to contain committed transactions) can also be used on join/recovery, as long as they are copied to the node's ``ledger.directory`` directory.
+
+Invariants
+----------
+
+1. To facilitate audit and verification of the integrity of the ledger, individual ledger files always end on a signature transaction.
+
+2. For operator convenience, all committed ledger files (``.committed`` suffix) are the same on all up-to-date nodes. More precisely, among up-to-date nodes:
+
+- Committed ledger files start and end at the same ``seqno``.
+- Committed ledger files with the same name are byte-identical.
+
+3. Snapshots are always taken at the ``seqno`` of a signature transaction.
```

Comment:
Maybe rephrase to "snapshots end at a signature", to make it clear that not every signature produces a snapshot?